package.json

```diff
@@ -8,10 +8,10 @@
   "scripts": {
     "release": "changeset publish",
     "prepare": "pnpm build",
-    "build": "pnpm --filter ember-headless-form build",
+    "build": "pnpm --filter ember-headless-form build && pnpm --filter '@ember-headless-form/*' build",
```

Comment:
This is a cheap workaround for dependency problems, as we need to build the main `ember-headless-form` first before the new one, because the latter consumes types of the former, which would lead to build/type errors when the `/dist` output containing the declaration files is not ready yet. This could be replace by a turborepo based setup in the future...

Reply:
does this repo need to do the `pnpm install --force` or `pnpm install` after a build to get around the dependenciesMeta.*.injected issue?

if not, we can set up turbo right away

Reply:
I don't use the injected thing at all so far, because... it worked without it 🤷‍♂️

Reply:
legit, I'll have a go at setting up turbo in here soon